spec/puppetserver/ca/sign_action_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'puppetserver/ca/sign_action'
+require 'puppetserver/ca/cli'
+require 'puppetserver/ca/logger'
+
+RSpec.describe 'Puppetserver::Ca::SignAction' do
+  let(:err)    { StringIO.new }
+  let(:inform) { StringIO.new }
```

Comment:
This is a mock stdout, and the above is stderr (or more shortly `err` and `out`). `Logger#inform` goes to stdout, but I don't think we should name this after the logging method.